barretenberg/cpp/src/barretenberg/honk/proof_system/combiner_example_gen.py

```diff
@@ -87,6 +87,12 @@ def get_extended_univariates(instances, row_idx):
         result = np.array(extend_one_entity(result))
         return result
 
+
```

Comment:
Should this be doing something?

Reply:
wops, i just realised i introduced it here, did not realise, sorry